grc/templates/birth-registration/dob.html

```diff
@@ -19,17 +19,13 @@
                 {{ form.csrf_token }}
 
                 <div class="govuk-form-group {{ 'govuk-form-group--error' if form.errors }}">
-                    <fieldset class="govuk-fieldset" role="group" aria-describedby="fieldset-hint">
+                    <fieldset class="govuk-fieldset" role="group">
                         <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
                             <h1 class="govuk-fieldset__heading">
                                 What is the date of birth on your birth or adoption certificate?
                             </h1>
                         </legend>
 
-                        <div class="govuk-hint" id="fieldset-hint">
-                            Enter the year in full - for example, 1987
-                        </div>
-
```

Comment:
Have we been asked to remove this?
I wonder if we discussed it, but forgot to add it to the Google Doc.
Can you check with Spence or Alex to see if they want to keep this?

Reply:
Tor confirmed it should stay